### PR TITLE
Run NFTs e2e tests on a controlled wallet address

### PR DESCRIPTION
**Pull request by michalinacienciala**
Previously we run tests by importing `bravonaver.eth` address as a read-only account. We don't control that address and can't guarantee that it's state won't change (changes may lead to unstable tests). Now we're switching to importing the third address of the `testertesting.eth` account (`0x6f1b1f1feb01235e15a7962f16c389c7f8218ed6`) as a read-only. We control this address and can decide on its state.

Latest build: [extension-builds-3487](https://github.com/tahowallet/extension/suites/13715479293/artifacts/758632877) (as of Mon, 19 Jun 2023 21:31:19 GMT).

### CODE REVIEW

**Comment by michalinacienciala** [comment]
I re-run the workflow with tests a couple of times to make sure it's not randomly failing - all executions were green: https://github.com/tahowallet/extension/actions/runs/5308837549.

File: e2e-tests/nfts.spec.ts [hunk 1]
```diff
@@ -36,10 +36,30 @@ test.describe("NFTs", () => {
         }
       })
 
-      await walletPageHelper.onboarding.addReadOnlyAccount("bravonaver.eth")
+      await walletPageHelper.onboarding.addReadOnlyAccount(
+        "0x6f1b1f1feb01235e15a7962f16c389c7f8218ed6"
```
Comment (jagodarybacka) [inline e2e-tests/nfts.spec.ts:40]: Do we plan to reuse that address for other e2e tests? Then we can make it a variable.

File: e2e-tests/nfts.spec.ts [hunk 2]
```diff
@@ -36,10 +36,30 @@ test.describe("NFTs", () => {
...
       await walletPageHelper.goToStartPage()
       await walletPageHelper.setViewportSize()
+
+      // Switch to Optimism, then to Polygon and then to Arbitrum (to load NFTs
+      // on those chains).
+      await page.getByTestId("top_menu_network_switcher").last().click()
+      await page.getByText(/^Optimism$/).click()
+      await expect(
+        page.getByTestId("top_menu_network_switcher").last()
+      ).toHaveText(/^Optimism$/)
```
Comment (jagodarybacka) [inline e2e-tests/nfts.spec.ts:52]: What do you think about making it a util function where you just specify what network you want to switch to?

**Review by jagodarybacka — DISMISSED** [review]
Works great, some ideas for improvements but non-blocking, lmk if you want to apply it here or later 😄

File: e2e-tests/nfts.spec.ts (hunk 1 above)
Comment (michalinacienciala) [reply e2e-tests/nfts.spec.ts:40]: I don't have a particular test in mind yet where we would reuse it. Let's leave it as it is for now - if we introduce the test which uses the same account, we can parametrize this then.

File: e2e-tests/nfts.spec.ts [hunk 3]
```diff
@@ -70,7 +90,9 @@ test.describe("NFTs", () => {
 
       await page
         .getByTestId("nft_account_filters")
-        .filter({ hasText: "bravonaver.eth" })
+        .filter({
+          hasText: /^(Phoenix|Matilda|Sirius|Topa|Atos|Sport|Lola|Foz)$/,
```
[the remark this answers is not in the document]
Comment (michalinacienciala) [reply e2e-tests/nfts.spec.ts:94]: Yeah, good point, displaying the label based on UNS/ENS is a functionality related to NFTs, so it makes sense to test it in this test. Will do.

File: e2e-tests/nfts.spec.ts (hunk 2 above)
Comment (michalinacienciala) [reply e2e-tests/nfts.spec.ts]: Done in 602e12bd9e2aaac9cdf34f84d90f72ff54364061.

File: e2e-tests/nfts.spec.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (michalinacienciala) [reply e2e-tests/nfts.spec.ts:83]: I thought I will be able to buy the domain myself, but had to ask Ops. Let's maybe not wait for that to get processed and merge the PR as it is... I'll update the account name in a separate PR once we have the domain secured - wdyt?

File: e2e-tests/nfts.spec.ts (hunk 1 above)
Comment (Shadowfiend) [reply e2e-tests/nfts.spec.ts:40]: We should put it in a named constant regardless—no magic strings please!

**Review by Shadowfiend — APPROVED** [review]
Reapproving because I did it too soon <_<

**Pull request merged**